react/week1/my-app/src/index.js

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import "./index.css";
+const root = document.getElementById("root");
+
+/* Todo List
+
+* Get out of bed, Wed Sep 13 2017
+* Brush teeth, Thu Sep 14 2017
+* Eat breakfast, Fri Sep 15 2017 */
+
+const todo = [
+  {
+    description : " Get out of bed",
+    deadline : " Wed Sep 13 2017"
+  },
+  {
+    description : "Brush teeth",
+    deadline : " Thu Sep 14 2017"
+  },
+
+  {
+    description : "Eat breakfast",
+    deadline : "Fri Sep 15 2017 "
+  }
+];
+class Head extends React.Component {
+  render() {
+
+    return <h1>Todo List</h1>
+  }
+
+}
+
+function MyTodo(props) {
+  const list = props.list;
+  const myToDoList = list.map(tasks => {
+    return (
```

Comment:
You could also have a component for the todos, so your final list component would look something like:

```jsx
function TodoList(props) {
   return (
     <ul>{props.todos.map(todo => <Todo description={...} deadLine={...} } />)}</ul>
   )
}
```